src/openforms/forms/api/viewsets.py

```diff
@@ -102,6 +103,34 @@ def get_serializer_context(self):
             )
         return context
 
+    @transaction.atomic()
+    def create(self, request, *args, **kwargs):
+        response = super().create(request, *args, **kwargs)
+
+        form = get_object_or_404(Form, uuid=self.kwargs["form_uuid_or_slug"])
+        transaction.on_commit(
+            lambda: create_form_variables_for_components.apply_async(
+                args=(form.id,),
+                countdown=60,
+            )
+        )
+
+        return response
+
+    @transaction.atomic()
+    def update(self, request, *args, **kwargs):
+        response = super().update(request, *args, **kwargs)
+
+        form = get_object_or_404(Form, uuid=self.kwargs["form_uuid_or_slug"])
+        transaction.on_commit(
+            lambda: create_form_variables_for_components.apply_async(
+                args=(form.id,),
+                countdown=60,
+            )
+        )
+
+        return response
```

Comment:
Would it make sense to implement this on the `FormStepSerializer.save` instead? That one calls out to `create/update`, so you'd only have to do it once, and there you have access to `self.instance.form` to look up the form ID.

Also - lambda's have late binding, it's safer to use `partial` instead.

Reply:
you're right, I'll move it